x11-libs/libxcb/libxcb-1.17.0.recipe

```diff
@@ -194,7 +196,7 @@ INSTALL()
 		$developDir \
 		$maybe_manDir_man3
```

Comment:
The conditional part can be removed since this isn't built for gcc2 any more. Otherwise, there is no documentation at all on x86.

Reply:
No, for x86_64, `man3` is moved to the devel package before the `rm`, so there it is included.